wake-ec2.yml

```diff
@@ -1,12 +1,14 @@
 ---
-
-# Change state of all instances in the EC2 environment
   - hosts: localhost
     tasks:
-      - name: Deploy ec2.py
-        file: src=ec2.py dest=./ec2.py mode=0744
+      - name: Copy ec2.py to /tmp
+        copy: src="./ec2.py" dest="/tmp/ec2.py" mode=0744
+      - name: Copy ec2.ini to /tmp
+        copy: src="./ec2.ini" dest="/tmp/ec2.ini" mode=0644
       - name: Get stopped instances
         shell: "./ec2.py --stopped --refresh-cache"
+        args:
+          chdir: /tmp
```

Comment:
I'm not sure about hardcoding `/tmp` and it all seems a bit complicated (compared to GCE).

How about instead resetting the cache with `./ec2.py --refresh-cache` afterwards?